packages/flutter_tools/lib/src/android/device_android.dart

```diff
@@ -213,12 +213,12 @@ class AndroidDevice extends Device {
       return false;
     }
     if (runCheckedSync(adbCommandForDevice(['shell', 'pm', 'path', app.id])) == '') {
-      logging.info(
+      printTrace(
```

Comment:
nit: while you're here, draw this all onto one line. We're only saving 6 characters here so the added ugliness of the newline isn't worth it.